src/constants.js

```diff
@@ -9,10 +9,10 @@ export const UNIT_MIB = 1024 ** 2;
 export const UNIT_GIB = 1024 ** 3;
 
 export const RELEASES = {
-  [RHEL_8]: 'Red Hat Enterprise Linux (RHEL) 8',
   [RHEL_9]: 'Red Hat Enterprise Linux (RHEL) 9',
-  'centos-8': 'CentOS Stream 8',
+  [RHEL_8]: 'Red Hat Enterprise Linux (RHEL) 8',
   'centos-9': 'CentOS Stream 9',
+  'centos-8': 'CentOS Stream 8',
```

Comment:
I'm not sure we should rely on the insertion order of keys in an object. I think theoretically for strings it should work, but most js implementations will reorder keys which are integers.

Reply:
https://github.com/tc39/proposal-for-in-order so as long as we use for...in there is a guaranteed ordering from 2020 it seems.

Reply:
`Map` objects preserve insertion order. I'm not sure if it would work as a drop in replacement for the current object but it might be a bit cleaner overall than trying to implement some funky sorting logic in the actual components.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Reply:
Hmm, will give the issue another look. I like the idea of changing the object to the map one, so I'll look into that. I'll switch this PR to draft for now.

Reply:
Alright, the RELEASES Object was converted to Map. When we want to add another release option in the future, the order in which we add it should be the very same as that which will be shown in the Release dropdown menu.